data_collection/web/guides.rst

```diff
@@ -10,3 +10,67 @@ Downloads and Outlinks
 ----------------------
 
 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop p
+
+Custom Events
+-------------
+
+Custom events enable tracking user actions that do not have dedicated methods in the existing tracker API, allowing web analysts to accurately measure and analyze any domain. Many integrations, including those offered by Tag Manager, use custom events for tracking actions detectable only on client-side, e.g. scrolling a page, viewing images, interacting with a video player, filling forms etc.
```

Comment:
Let's stick to one version and refer to the person viewing the website (whose actions are being tracked) as `visitor`.

There were some occurrences of the `user` in the previous PRs. Please check your PRs against the `user` and replace it with the `visitor`

```suggestion
Custom events enable tracking of the visitor actions that do not have dedicated methods in the existing tracker API, allowing web analysts to accurately measure and analyze any domain. Many integrations, including those offered by Tag Manager, use custom events for tracking actions detectable only on client-side, e.g. scrolling a page, viewing images, interacting with a video player, filling forms etc.
```